system/ee/ExpressionEngine/Controller/Utilities/SyncConditionalFields.php

```diff
@@ -237,7 +237,7 @@ public function evaluate()
             'message_type' => 'success',
             'channel_id' => $channel_id,
             'entries' => $entries->pluck('entry_id'),
-            'entries_proccessed' => $entries->count()
+            'entries_processed' => $entries->count()
```

Comment:
We need to confirm through the core that the updated array key is used. @matthewjohns0n

Reply:
I did verify this one and made sure it was updated everywhere.